src/gui/mzroll/spectrawidget.cpp

```diff
@@ -218,13 +218,15 @@ void SpectraWidget::overlayPeptideFragmentation(QString peptideSeq,MassCutoff *p
 
 void SpectraWidget::overlayPeakGroup(PeakGroup* group)
 {
-    _currentGroup = nullptr;
+    delete _currentGroup;
```

Comment:
`_currentGroup` is a `PeakGroup` on stack now. We cannot `delete` it. This statement causes compilation failure.

Reply:
A proper example for why last-minute changes are dangerous :)

I'll remove this.